e2e/tests/workspace/cdr-version-display.spec.ts

```diff
@@ -10,17 +10,6 @@ describe('Workspace CDR Version display', () => {
         await signIn(page);
     });
 
-    test('The CDR version displays in the workspace navigation bar', async () => {
```

Comment:
checking cdr version on Data page is moved to `workspace-create.spec` test (one less new workspace).